docs/management/managing-saved-objects.asciidoc

```diff
@@ -1,81 +1,82 @@
 [[managing-saved-objects]]
-== Managing Saved Objects
+== Saved objects
```

Comment:
nit: we could rename the file and related image file to remove `management` in its name.